src/transformers/models/openai/configuration_openai.py

```diff
@@ -136,7 +136,6 @@ def __init__(
         summary_activation=None,
         summary_proj_to_labels=True,
         summary_first_dropout=0.1,
-        use_cache=True,
```

Comment:
openai actually has no `use_cache` logic implemented so deleted it from config...